projectmanagement/src/main/java/life/qbic/projectmanagement/domain/project/sample/SampleRegistrationRequest.java

```diff
@@ -21,4 +22,14 @@ public record SampleRegistrationRequest(String label, BatchId assignedBatch,
                                         BiologicalReplicateId replicateReference,
                                         SampleOrigin sampleOrigin) {
 
+  public SampleRegistrationRequest(String label, BatchId assignedBatch, ExperimentId experimentId,
```

Comment:
As those are a lot of parameters, you can use records compact constructor and simply remove the (parameters)
https://blogs.oracle.com/javamagazine/post/java-record-compact-canonical-constructor